lib/pagseguro/company.rb

```diff
@@ -0,0 +1,41 @@
+module PagSeguro
+  class Company
+    include Extensions::MassAssignment
+    include Extensions::EnsureType
+
+    attr_accessor :name
+    attr_accessor :display_name
+    attr_accessor :website_url
+
+    attr_reader :address
+    attr_reader :partner
+
+    def address=(address)
+      @address = ensure_type(Address, address)
+    end
+
+    def partner=(partner)
+      @partner = ensure_type(Partner, partner)
+    end
+
+    def phones
+      @phones ||= Phones.new
+    end
+
+    def phones=(_phones=[])
+      _phones.each do |phone|
```

Comment:
Maybe we can use `map` here:

```
_phones.map do |phone|
    ensure_type(Phone, phone)
end
```

but Don't change anything I will take notes to central ;)